sources/objectives/minimum_dissipation_objective_function.f90

```diff
@@ -226,6 +233,10 @@ subroutine minimum_dissipation_objective_function_compute(this, design, fluid)
     call field_addcol3(objective_field, wo2, wo2)
     call field_addcol3(objective_field, wo3, wo3)
 
+    if (this%if_mask) then
+       call mask_exterior_const(objective_field, this%mask, 0.0_rp)
+    end if
```

Comment:
We should probably not spend the time computing all those gradients above if you
just 0 it anyway.